vllm/config/cache.py

```diff
@@ -90,8 +90,9 @@ class CacheConfig:
     mamba_page_size_padded: int | None = None
     """ Optional override for mamba page size; used by hybrid mamba/attention
     models to ensure exact alignment with attention page size."""
-    mamba_block_size: int | None = None
-    """Size of a contiguous cache block in number of tokens for mamba cache."""
+    mamba_block_size: int | None = Field(default=None, gt=0, multiple_of=256)
```

Comment:
do we have to force it to be multiple of 256? I think this limitation should be verified by each mamba attention backend.